resources/views/contests/partials/_form_contest.blade.php

```diff
@@ -4,12 +4,12 @@
 
 <div class="form-item -padded">
     <label class="field-label" for="signup_start_date">Signup Start Date:</label>
-    <input type="date" name="signup_start_date" id="signup_start_date" value={{ $contest->waitingRoom->signup_start_date or old($contest->waitingRoom->signup_start_date)}} class="text-field"></input>
+    <input type="date" name="signup_start_date" id="signup_start_date" value={{ $contest->waitingRoom->signup_start_date or 'MM/DD/YYYY' }} class="text-field"></input>
```

Comment:
I do a similar approach in the user create form... uses the same partial as the edit form :dancer: